charts/warden/charts/admission/templates/deployment.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.global.admission.enabled }}
```

Comment:
I would suggest disabling/enabling the whole sub-chart. It would be much easier to maintain and more readable.

[Here](https://github.com/kyma-project/serverless/blob/main/config/serverless/Chart.yaml#L7-L11) is a small example of how we did it in serverless.

I performed a small experiment and it needs a few changes in a few places. Works fine:

```diff
diff --git a/charts/warden/Chart.yaml b/charts/warden/Chart.yaml
index 307f249..ef0984d 100644
--- a/charts/warden/Chart.yaml
+++ b/charts/warden/Chart.yaml
@@ -23,3 +23,8 @@ version: 0.0.6
 # It is recommended to use it with quotes.
 appVersion: "0.0.6"
 
+dependencies:
+  - name: warden-admission
+    condition: admission.enabled
+  - name: warden-operator
+    condition: operator.enabled
diff --git a/charts/warden/values.yaml b/charts/warden/values.yaml
index a252e24..d98918a 100644
--- a/charts/warden/values.yaml
+++ b/charts/warden/values.yaml
@@ -8,6 +8,12 @@ imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
 
+admission:
+  enabled: true
+
+operator:
+  enabled: true
+
 #Service configuration
 global:
   name: warden
```